contracts/modules/delegates/IexecPoco1Delegate.sol

```diff
@@ -25,7 +26,13 @@ struct Matching {
     bool hasDataset;
 }
 
-contract IexecPoco1Delegate is IexecPoco1, DelegateBase, IexecEscrow, SignatureVerifier {
+contract IexecPoco1Delegate is
+    IexecPoco1,
+    DelegateBase,
+    IexecEscrow,
+    SignatureVerifier,
+    IexecOrderManagementDelegate
```

Comment:
Use interface instead

Reply:
Using the interffaces I should change the restriction of the `_computeVolume` from internal to external ?

Reply:
You right I see!